src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -192,7 +192,8 @@ var ReactElementValidator = {
       validType,
       'React.createElement: type should not be null, undefined, boolean, or ' +
         'number. It should be a string (for DOM elements) or a ReactClass ' +
-        '(for composite components).%s',
+        '(for composite components). See ' +
+        'https://facebook.github.io/react/warning/create-element-types.html%s',
```

Comment:
I haven't read the actual proposal but assuming we move forward, 2 things:

1. Please use a short url, we can hook that up later. Let's say `fb.me/react-warning-create-element` or something similar.
2. The url should be the last thing in the message so we should have the `%s` before it.